src/vec_buffers.h

```diff
@@ -42,22 +42,22 @@
 namespace quad {
 namespace vec {
 
-/* Available cases of allocating memory
- * NO_ALLOC:    do not allocate any memory
- * SLICE_ALLOC: allocate only a vector of pointers each points to an allocated
+/** Available cases of allocating memory
```

Comment:
In fact, you can also comment enum values with Doxygen.

```C++
/// Memory allocation strategy.
enum class BufMemAlloc {
    /// Do not allocate any memory.
    NONE = 0,

    /// Allocate only a vector of pointers each points to an allocated memory.
    SLICE,

    /// Allocate a zero buffers and push back to the vector memory.
    ZERO_EXTEND,

    /// Allocate only a vector of pointers that are from two vectors of two
    /// input buffers vector.
    COMBINED,

    /// Fully allocate memory including a vector of pointers each for a memory
    /// of `size` elements.
    FULL,
};
```

![doxy](https://user-images.githubusercontent.com/34654278/43307940-24ddbc2c-9180-11e8-946e-79a37d8834bc.png)